pkg/koordlet/util/runtime/handler/crio_runtime_test.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handler
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/prashantv/gostub"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+
+	mockclient "github.com/koordinator-sh/koordinator/pkg/koordlet/util/runtime/handler/mockclient"
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/util/system"
+)
+
+func Test_NewCrioRuntimeHandler(t *testing.T) {
+	stubs := gostub.Stub(&GrpcDial, func(context context.Context, target string, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
+		return &grpc.ClientConn{}, nil
+	})
+	defer stubs.Reset()
+
+	helper := system.NewFileTestUtil(t)
+	defer helper.Cleanup()
+
+	helper.WriteFileContents("/var/run/crio/crio.sock", "test")
+	system.Conf.VarRunRootDir = filepath.Join(helper.TempDir, "/var/run")
```

Comment:
Perhaps there is no need to change the VarRunRootDir since it is already set in the `system.NewFileTestUtil(t)`.

Reply:
Hi saintube .Thank you for your feedback and suggestions. You are right, this directory is set there system.NewFileTestUtil(t) , and this test file follows the best practice of test cases similar to the files of containerd_runtime_test.go , I guess it may be to make the test results displayed better.